sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerPage.scala

```diff
@@ -206,18 +208,34 @@ private[ui] class SqlStatsPagedTable(
 
   override def headers: Seq[Node] = {
     val sqlTableHeadersAndTooltips: Seq[(String, Boolean, Option[String])] =
-      Seq(
-        ("User", true, None),
-        ("JobID", true, None),
-        ("GroupID", true, None),
-        ("Start Time", true, None),
-        ("Finish Time", true, Some(THRIFT_SERVER_FINISH_TIME)),
-        ("Close Time", true, Some(THRIFT_SERVER_CLOSE_TIME)),
-        ("Execution Time", true, Some(THRIFT_SERVER_EXECUTION)),
-        ("Duration", true, Some(THRIFT_SERVER_DURATION)),
-        ("Statement", true, None),
-        ("State", true, None),
-        ("Detail", true, None))
+      if (showSessionLink) {
+        Seq(
+          ("User", true, None),
+          ("JobID", true, None),
+          ("GroupID", true, None),
+          ("Session ID", true, None),
+          ("Start Time", true, None),
+          ("Finish Time", true, Some(THRIFT_SERVER_FINISH_TIME)),
+          ("Close Time", true, Some(THRIFT_SERVER_CLOSE_TIME)),
+          ("Execution Time", true, Some(THRIFT_SERVER_EXECUTION)),
+          ("Duration", true, Some(THRIFT_SERVER_DURATION)),
+          ("Statement", true, None),
+          ("State", true, None),
+          ("Detail", true, None))
+      } else {
+        Seq(
+          ("User", true, None),
+          ("JobID", true, None),
+          ("GroupID", true, None),
+          ("Start Time", true, None),
+          ("Finish Time", true, Some(THRIFT_SERVER_FINISH_TIME)),
+          ("Close Time", true, Some(THRIFT_SERVER_CLOSE_TIME)),
+          ("Execution Time", true, Some(THRIFT_SERVER_EXECUTION)),
+          ("Duration", true, Some(THRIFT_SERVER_DURATION)),
+          ("Statement", true, None),
+          ("State", true, None),
+          ("Detail", true, None))
```

Comment:
The code in the `if` block and the `else` block are almost same except for `("Session ID", true, None)`.
Could you refactor the code?